Gemfile

```diff
@@ -12,6 +12,8 @@ group :development do
   gem 'rubocop',       '~> 0.37.0'
   gem 'yard',          '~> 0.8.7'
   gem 'simplecov',     '~> 0.11.1'
+  gem 'mutant',        '~> 0.8.10'
```

Comment:
Good point. I wanted to be super-explicit here but on second thought I realize this is questionable, I'll remove this line.